ESS/v3/_summit.yaml

```diff
@@ -97,3 +97,34 @@ instances:
                 - AIN1
               offset: 0
               scale: 344738.0
+  - sal_index: 202
+    data_clients:
+      - client_class: LabJackAccelerometerDataClient
+        # LabJack reading three accelerometers
+        config:
+          device_type: T7
+          connection_type: WIFI
+          identifier: aux-labjackwifi.cp.lsst.org
+          min_frequency: 0
+          max_frequency: 100
+          num_frequencies: 200
+          accelerometers:
+            - sensor_name: AuxTel-M2
+              location: AuxTel M2 x=el y=az
+              analog_inputs: [1, 2, 3]
+              offsets: [2.49017, 2.44424, 2.44589]
+              # published scales: [989.58, 985.72, 999.46] mV/g
+              scales: [0.100978, 0.100584, 0.101986]
+            - sensor_name: AuxTel-Truss
+              location: AuxTell Truss x=el y=az
```

Comment:
The Truss unit is turned 90 degrees with respect to the other two, so this should read:
x=el z=az

Reply:
Thank you for catching that! I changed it and added a comment as well (to reinforce the value for maintainers of the yaml file)